sample-configs/collector-config-advanced.yaml

```diff
@@ -0,0 +1,297 @@
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: my-collector
```

Comment:
Should we have a different name for each of these, in case the user applies them all and expects to see all of them together?

Reply:
Yeah, I can change this, there's no harm in having the names be different from each other